translations/messages+intl-icu.de.yml

```diff
@@ -1352,6 +1357,7 @@ explanation.export.docx: "Es werden die Daten entsprechend der Auswahl in den Gr
 explanation.export.potentialAreas: "Es werden die Begründungen zu den Stellungnahmen mit Potenzialflächen exportiert."
 explanation.export.projectlogin: "Bitte melden Sie sich bei {projectName} an, um das Archiv herunterzuladen."
 explanation.export.statements: "Exportiert ausgewählte Stellungnahmen mit allen Daten als XLSX."
+explanation.export.statements.zip: "Exportiert ausgewählte Stellungnahmen mit allen Daten und Anhängen als ZIP-Archiv für den Import in das Einwendungsmanagement (EWM)."
```

Comment:
you should name things consistently. Here it is called "ZIP-Archiv", at other places "zip"

Reply:
Changed it to ZIP-Archiv except for the tab name in [9f8a2ef](https://github.com/demos-europe/demosplan-core/pull/2337/commits/9f8a2efe24a649667e76190d9d2c369767da6047). Discussion see here https://demosdeutschland.slack.com/archives/C01HRQ2CC1L/p1700555377203789